src/main/java/gregtech/integration/theoneprobe/provider/CoverProvider.java

```diff
@@ -0,0 +1,233 @@
+     * @param voiding   the voiding cover to get data from
+     */
+    private static void itemVoidingInfo(@Nonnull IProbeInfo probeInfo, @Nonnull CoverItemVoiding voiding) {
+        String unit = " {*gregtech.top.unit.items*}";
+
+        ItemFilterContainer container = voiding.getItemFilterContainer();
+        if (voiding instanceof CoverItemVoidingAdvanced) {
+            CoverItemVoidingAdvanced advanced = (CoverItemVoidingAdvanced) voiding;
+            VoidingMode mode = advanced.getVoidingMode();
+            voidingText(probeInfo, mode, unit, container.getTransferStackSize());
+        }
+        itemFilterText(probeInfo, container.getFilterWrapper().getItemFilter());
+    }
+
+    /**
+     * Displays text for {@link CoverPump} related covers
+     *
+     * @param probeInfo the info to add the text to
+     * @param pump      the pump to get data from
+     */
+    private static void pumpInfo(@Nonnull IProbeInfo probeInfo, @Nonnull CoverPump pump) {
+        String rateUnit = IProbeInfo.STARTLOC + pump.getBucketMode().getName() + IProbeInfo.ENDLOC;
+
+        if (pump instanceof CoverFluidVoiding) {
+            fluidVoidingInfo(probeInfo, (CoverFluidVoiding) pump);
+        }
+
+        // do not display the regular rate if the cover has a specialized rate
+        if (!(pump instanceof CoverFluidVoiding) && (!(pump instanceof CoverFluidRegulator) || ((CoverFluidRegulator) pump).getTransferMode() == TransferMode.TRANSFER_ANY)) {
+            transferRateText(probeInfo, pump.getPumpMode(), " " + rateUnit, pump.getBucketMode() == CoverPump.BucketMode.BUCKET ? pump.getTransferRate() / 1000 : pump.getTransferRate());
+        }
+
+        FluidFilterContainer filter = pump.getFluidFilterContainer();
+        if (pump instanceof CoverFluidRegulator) {
+            CoverFluidRegulator regulator = (CoverFluidRegulator) pump;
+            transferModeText(probeInfo, regulator.getTransferMode(), rateUnit, regulator.getTransferAmount(), filter.getFilterWrapper().getFluidFilter() != null);
+        }
+        fluidFilterText(probeInfo, filter.getFilterWrapper().getFluidFilter());
+    }
+
+    /**
+     * Displays info for {@link CoverFluidVoiding} related covers
+     *
+     * @param probeInfo the info to add the text to
+     * @param voiding   the voiding cover to get data from
+     */
+    private static void fluidVoidingInfo(@Nonnull IProbeInfo probeInfo, @Nonnull CoverFluidVoiding voiding) {
+        String unit = voiding.getBucketMode() == CoverPump.BucketMode.BUCKET ? " {*gregtech.top.unit.fluid_buckets*}" : " {*gregtech.top.unit.fluid_milibuckets*}";
+
+        if (voiding instanceof CoverFluidVoidingAdvanced) {
+            CoverFluidVoidingAdvanced advanced = (CoverFluidVoidingAdvanced) voiding;
+            VoidingMode mode = advanced.getVoidingMode();
+            voidingText(probeInfo, mode, unit, voiding.getBucketMode() == CoverPump.BucketMode.BUCKET ? advanced.getTransferAmount() / 1000 : advanced.getTransferAmount());
```

Comment:
This displays the voiding amount in Void Overflow mode, even  when there is a filter in the voiding cover, which overrides this amount.